spyder/widgets/ipythonconsole/shell.py

```diff
@@ -70,6 +71,9 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         # To save kernel replies in silent execution
         self._kernel_reply = None
 
+        # To set the color of the matched parentheses
+        self.set_bracket_matcher_color_scheme(self.syntax_style)
```

Comment:
Why is it necessary to add this here? I mean, the call to `set_bracket_matcher_color_scheme` in `set_color_scheme` should be enough, shouldn't it?